examples/todo/src/models/todo.model.ts

```diff
@@ -27,22 +27,18 @@ export class Todo extends Entity {
   @property({
     type: 'boolean',
   })
-  isComplete: boolean;
+  isComplete?: boolean;
 
   @property({
     type: 'string',
   })
-  remindAtAddress: string; // address,city,zipcode
+  remindAtAddress?: string; // address,city,zipcode
 
   // TODO(bajtos) Use LoopBack's GeoPoint type here
   @property({
     type: 'string',
   })
-  remindAtGeo: string; // latitude,longitude
-
-  getId() {
```

Comment:
What happened to this function - are you sure we don't need it any more?

Reply:
Should we perhaps modify the template used by the CLI tool to always include `getId()` implementation, preferably after the `constructor` function?

Reply:
I think with the refactor my comment on GitHub got lost somewhere ... but the reason for me to not include the `getId` function is as follows:

- The underlying `Entity` class actually implements it based on the metadata ... so any property can be the `id` field and it handles it correctly (and while we can do this with the CLI ... I don't see the point of overriding the logic from the base class. 

Here's the reference code: https://github.com/strongloop/loopback-next/blob/08c2d896367a7f8ecc5153578f2a698eefdea7a1/packages/repository/src/model.ts#L218-L231

Reply:
Good point. I guess my concern is about performance - an explicit `getId` implementation returning `this.id` is surely faster than the generic solution processing model definition.

But as Donald Knuth said, _premature optimization is the root of all evil_. I am ok with your proposed change, we can always add explicit `getId()` implementation later, when there is a legitimate need.